lib/matplotlib/tests/test_determinism.py

```diff
@@ -50,6 +83,50 @@ def _save_figure(objects='mhi', fmt="pdf", usetex=False):
         A = [[2, 3, 1], [1, 2, 3], [2, 1, 3]]
         fig.add_subplot(1, 6, 5).imshow(A, interpolation='bicubic')
 
+    if 'p' in objects:
+        # add a polar projection
+        px = fig.add_subplot(projection="polar")
+        pimg = px.imshow([[2]])
+        pimg.set_clip_path(Circle((0, 1), radius=0.3333))
+
+        # add a text-based clipping path (origin: demo_text_path.py)
+        (ax1, ax2) = fig.subplots(2)
+        arr = plt.imread(get_sample_data("grace_hopper.jpg"))
+        text_path = TextPath((0, 0), "!?", size=150)
+        p = PathClippedImagePatch(text_path, arr, ec="k")
+        offsetbox = AuxTransformBox(IdentityTransform())
+        offsetbox.add_artist(p)
+        ao = AnchoredOffsetbox(loc='upper left', child=offsetbox, frameon=True,
+                               borderpad=0.2)
+        ax1.add_artist(ao)
+
+        # add a 2x2 grid of path-clipped axes (origin: test_artist.py)
+        exterior = Path.unit_rectangle().deepcopy()
+        exterior.vertices *= 4
+        exterior.vertices -= 2
+        interior = Path.unit_circle().deepcopy()
+        interior.vertices = interior.vertices[::-1]
+        clip_path = Path.make_compound_path(exterior, interior)
+
+        star = Path.unit_regular_star(6).deepcopy()
+        star.vertices *= 2.6
+
+        fig, (row1, row2) = plt.subplots(2, 2, sharex=True, sharey=True)
```

Comment:
This makes a second new figure.

Reply:
Argh, thanks - yep.  I did confirm that it fails without the fix in place, but I didn't notice the variable shadowing.